src/VisualQueryBuilder/components/LabelFilterItem.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   defaultOp: string;
   item: Partial<QueryBuilderLabelFilter>;
   items: Array<Partial<QueryBuilderLabelFilter>>;
-  onChange: (value: QueryBuilderLabelFilter) => void;
+  onChange: (value: Partial<QueryBuilderLabelFilter>) => void;
```

Comment:
I can't really see why it's Partial for all item, items, and onChange.

Reply:
Can you elaborate? Not sure I understand.

Reply:
I see at the bottom we call `onChange` sometimes without value, and sometimes without value. Again not sure why, but that explains the partial.